javascript/packages/fury/lib/reader.ts

```diff
@@ -99,6 +99,13 @@ export const BinaryReader = (config: Config) => {
     return result;
   }
 
+  function stringUtf8OfVarint32() {
```

Comment:
Perhaps we should rename it to stringOfVarInt32, as it corresponds to the stringOfVarInt32 method in the writer. This change would also correct the naming mistake I made a long time ago.

Reply:
Accordingly, readVarInt32 in `packages/fury/lib/internalSerializer/string.ts` should also be moved to this function